src/components/atoms/InputField/InputField.tsx

```diff
@@ -61,9 +61,20 @@ const InputField: React.ForwardRefRenderFunction<
 
   const inputClassNames = classNames("input-field__input", inputClassName);
 
+  function keyDown(e: React.KeyboardEvent<HTMLInputElement>) {
+    if (e.key === "Escape" && e.target) {
+      e.currentTarget.blur();
+    }
+  }
```

Comment:
I wonder if we would also want to use `mousetrap` here for consistency?

Also, I wonder if we should be adding this functionality to our generic `InputField`, as opposed to just within the navbar search/similar 'focussed place' that the feature is desired for.

@mike-lvov Any thoughts?

Reply:
That is a solution @soichih came up with since for some reason mousetrap (as described in TODOs on top, and you asked above) for `s` was not causing any "action", while did just fine for other cases. 
I wonder if it also anyhow relates to fresh issue I observed with current state as of a05339aef51ae3551571ba21eb801118bb79e945: if go to "Direct messages" and then back to "Party chat" - `c` no longer toggles chat until I toggle by mouse or go back to "Direct messages"

Reply:
Does it relate to the fact that ‘S’ is already bound to movement in usekeyboardControls()?

Reply:
I would have to look at the code to be sure, but guessing the ‘no longer toggles’ is possibly going to relate to having the wrong function reference bound to mousetrap or similar, that only sets the state a certain way or similar (not on a computer to check at the moment)

Reply:
Based on another comment thread on this PR, I believe the issue/solution was that the key was already bound in mousetrap.